packages/compiler/src/render3/partial/api.ts

```diff
@@ -367,6 +367,48 @@ export enum FactoryTarget {
   NgModule = 4,
 }
 
+/**
+ * Describes the shape of the object that the `ɵɵngDeclareInjectable()` function accepts.
+ *
+ * This interface serves primarily as documentation, as conformance to this interface is not
+ * enforced during linking.
+ */
+export interface R3DeclareInjectableMetadata extends R3PartialDeclaration {
+  /**
+   * If provided, specifies that the declared injectable belongs to a particular injector:
+   * - `InjectorType` such as `NgModule`,
+   * - `'root'` the root injector
+   * - `'any'` all injectors.
+   * If not provided, then it does not belong to any injector. Must be explicitly listed in the
+   * providers of an injector.
+   */
+  providedIn?: o.Expression;
+  /**
```

Comment:
nit: may be add a new line after the field and before the next field's comment (for slightly better readability and consistency with other interfaces format)?
```suggestion
  providedIn?: o.Expression;
  
  /**
```